app/models/status.rb

```diff
@@ -141,6 +143,11 @@ def non_sensitive_with_media?
     !sensitive? && media_attachments.any?
   end
 
+  before_validation :prepare_contents
+  before_create     :set_reblog
+  before_create     :set_visibility
+  before_create     :set_conversation
```

Comment:
No, I've split up a before_validation method into smaller methods, there is no change in the logic.